guides/17_Migration_to_3_0.md

```diff
@@ -53,4 +58,73 @@ handle them please remove.
 - AudioCallEnded
 - TURNMeetingEnded
 - StateMachineTransitionFailed
-- AudioDeviceSwitched
\ No newline at end of file
+- AudioDeviceSwitched
+
+### Updating `EventController` use cases
+We have de-coupled the `EventController` from `AudioVideoController`. Check below for the new changes and if updates are needed for your implementation.
+
+Use case 1: Update implementation of custom `EventController`
+```typescript
+interface EventController {
+  // Adds an observer for event published to this controller.
+  addObserver(observer: EventObserver): void;
+
+  // Removes an observer for event published to this controller.
+  removeObserver(observer: EventObserver): void;
+
+  // EventReporter that the EventController uses to send events to the Amazon Chime backend.
+  readonly eventReporter?: EventReporter;
+
+  // pushMeetingState has been deprecated
```

Comment:
Since we are removing we can say pushMeetingState removed over deprecated as SDK was not internally using it.

Reply:
While it was meant to be internal it was never marked as internal meaning some builder might have been using it, so I'd say it's safer to say deprecated